node/consensus/src/lib.rs

```diff
@@ -53,10 +53,12 @@ use tokio::{
     task::JoinHandle,
 };
 
-/// Percentage of mempool transactions capacity reserved for deployments.
-const CAPACITY_FOR_DEPLOYMENTS: usize = 20;
-/// Percentage of mempool transactions capacity reserved for executions.
-const CAPACITY_FOR_EXECUTIONS: usize = 80;
+/// The capacity of the mempool reserved for deployments.
+const CAPACITY_FOR_DEPLOYMENTS: usize = 1 << 8;
```

Comment:
The use of this constant further below (line 302) is now broken. 

To avoid missing it in the future, I suggest we introduce a new constant, e.g. `DEPLOYMENT_RATE`. This constant ensures that for every 10 transactions we are going to verify, at least 80% of those will be executions, preventing DoS from bogus deployments.

[In this PR](https://github.com/AleoHQ/snarkOS/pull/3089) a DEPLOYMENT_RATE of 20 was approved, but we can in the future go up to 50 to enhance deployment throughput.